eng/targetingpacks.targets

```diff
@@ -129,12 +129,14 @@
   <Target Name="UpdateLocalTargetingAndRuntimePack"
           Condition="'$(UseLocalTargetingRuntimePack)' == 'true'"
           AfterTargets="ResolveFrameworkReferences">
+    <Error Text="The shared framework must be built before the local targeting pack can be consumed."
+           Condition="!Exists('$(MicrosoftNetCoreAppRefPackDir)data\FrameworkList.xml')" />
```

Comment:
Microsoft.NET.WebAssembly.Webcil.csproj unintentionally built against the LKG even though `UseLocalTargetingRuntimePack` is true. Adding the error makes this more explicit.